public/js/load_comment.js

```diff
@@ -0,0 +1,99 @@
+
+//const & var
+const addComment = (data)=>{
+  try
+  {
+    const componentComments = document.createElement("div");
+    obj = JSON.parse(data);
+    console.log(obj);
+    if(obj.status === "noComment")
+    {
+      // TODO replace by innerHTML in message noComment (create the div in twig)
+      alert("Pas de commentaires supplémentaires trouvés."); 
+    }
+    else if(obj.status === "error")
+    {
+      // TODO replace by innerHTML in message error (create the div in twig)
+      message = "Une erreur innatendue est survenue:"+ obj.code + " " + obj.message;
+      alert(message);
+    }
+    else if(obj.status === "success")
+    {
+      comments = JSON.parse(obj.data);
+      comments.forEach(comment => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/soutenance14/ST/pullRequest?prid=8161867)
```suggestion
      comments.forEach((comment) => {
```